config/stored_requests_test.go

```diff
@@ -41,6 +43,26 @@ func TestQueryMakerNegative(t *testing.T) {
 	assertStringsEqual(t, query, expected)
 }
 
+func TestNormalResponseQueryMaker(t *testing.T) {
+	madeQuery := buildQueryResponses(sampleResponsesQueryTemplate, 1)
+	assertStringsEqual(t, madeQuery, "SELECT id, responseData, 'response' as type FROM stored_responses WHERE id in ($1)")
+}
+func TestResponseQueryMakerManyIds(t *testing.T) {
+	madeQuery := buildQueryResponses(sampleResponsesQueryTemplate, 11)
+	assertStringsEqual(t, madeQuery, "SELECT id, responseData, 'response' as type FROM stored_responses WHERE id in ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11)")
+}
+
+func TestResponsesQueryMakerNoIds(t *testing.T) {
+	madeQuery := buildQueryResponses(sampleResponsesQueryTemplate, 0)
+	assertStringsEqual(t, madeQuery, "SELECT id, responseData, 'response' as type FROM stored_responses WHERE id in (NULL)")
+}
+
+func TestResponseQueryMakerNegative(t *testing.T) {
+	query := buildQueryResponses(sampleResponsesQueryTemplate, -2)
+	expected := buildQueryResponses(sampleResponsesQueryTemplate, 0)
+	assertStringsEqual(t, query, expected)
+}
+
```

Comment:
This might be a nitpick, but I see these tests from line 46-64, are all testing the same function but with different cases (i.e. No IDs vs Many IDs). This can be refactored to be one table driven test with many test cases within it, which is something I prefer, but of course if you'd like to keep it as it is then no worries.

Reply:
Right, I followed the existing pattern, but yes, agree, this should be a table drive test. Updated.